src/app/pages/NFTByFtxId/Main/Main.module.scss

```diff
@@ -35,17 +40,41 @@
       text-align: left;
 
       @media screen and (max-width: $md-width) {
-       margin-top: 1rem;
+        margin-top: 1rem;
       }
 
       &Text {
-        padding-bottom: 2rem;
-        padding-top: 1rem;
+        padding: 1rem 0;
       }
 
       &Title {
         color: $orange;
       }
     }
   }
+
+  &__heroleteItem {
+    display: flex;
+    justify-content: space-around;
+    margin: 0.5rem 0;
+    text-align: center;
+
+    &Info {
+      color: $orange;
+      font-weight: 700;
+      font-size: 1.5rem;
+    }
+
+    &Price {
+      display: flex;
+      justify-content: space-between;
```

Comment:
I think would be best for the design to use flex start instead.

Reply:
The design indicates that should be separated. Otherwise a lot of visual weight would be on the left side and nothing and nothing to counterweight on the right side.
![image](https://user-images.githubusercontent.com/24862956/155710243-750b976e-4bf7-4a29-926b-dd45e720c3e9.png)